src/openMVG/localization/optimization.hpp

```diff
@@ -29,14 +29,22 @@ namespace localization{
  * the optical distortion, setting it to 0. 
  * @param[in] b_refine_pose Whether to refine the camera poses.
  * @param[in] b_refine_structure Whether to refine the 3D points.
+ * @param[in] outputFilename If not empty, a filename (possibly with path) without 
+ * extension where to save the scene before and after the bundle adjustment. The 
+ * file will be named outputFilename.BEFORE.json and outputFilename.AFTER.json.
+ * @param[in] minPointVisibility if > 0 it allows to use only the 3D points that 
+ * are seen in at least \p minPointVisibility views/frames, all the other 
+ * points (and associated 2D features) will be discarded.
  * @return true if the bundle adjustment has success.
  */
 bool refineSequence(std::vector<LocalizationResult> & vec_localizationResult,
                     bool allTheSameIntrinsics = true,
                     bool b_refine_intrinsic = true,
```

Comment:
We can now use the enum [BA_Refine](https://github.com/poparteu/openMVG/blob/popart_develop/src/openMVG/sfm/sfm_data_BA.hpp#L15).